eZ/Bundle/EzPublishRestBundle/Features/ContentTypeGroup/list.feature

```diff
@@ -22,7 +22,18 @@ Feature: Read all Content Type Groups
             | dif3rent   |
             | id_nt.fier |
 
-    Scenario: Can't read any Content Type Group without an authorized user
+    Scenario: List Content Type Groups without an authorized user
         Given I don't have permissions
+        And there are the following Content Type Groups:
+            | groups     |
+            | some       |
+            | dif3rent   |
+            | id_nt.fier |
         When I get Content Type Groups list
-        Then response has a not authorized error
+        Then response status code is 200
+        And response status message is "OK"
+        And response contains the following Content Type Groups:
+            | groups     |
+            | some       |
+            | dif3rent   |
+            | id_nt.fier |
```

Comment:
I've checked, and it's been like that since 5.0, I think. Controller actions that modify content types check the matching policies, but there isn't anything when reading content type groups (or content types, as far as I can see).

The thing is that I don't even think we have a policy for that.